src/libraries/System.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/JSSynchronizationContext.cs

```diff
@@ -21,7 +23,7 @@ internal sealed class JSSynchronizationContext : SynchronizationContext
     {
         private readonly Action _DataIsAvailable;// don't allocate Action on each call to UnsafeOnCompleted
         public readonly Thread TargetThread;
-        public readonly IntPtr TargetThreadId;
+        public readonly IntPtr TargetTID;
```

Comment:
What is the motivation here? IMO the original name was easier to read

Reply:
Mono code uses `tid` consistently for thread id of this type. That's `MonoNativeThreadId` or `pthread_t`

As opposed to `Thread.CurrentThread.ManagedThreadId` for example. I wanted to separate them clearly.